pkg/kubelet/cm/cpumanager/cpu_assignment.go

```diff
@@ -118,6 +120,23 @@ func (n *numaFirst) takeFullSecondLevel() {
 	n.acc.takeFullSockets()
 }
 
+// In Split L3 Topology, we take from the sets of uncorecache as the third level
```

Comment:
let's make a pass to make sure the terminology in the implementation PR match what we used in the KEP, the option name is the prime example but we should also review therms like uncore, L3/LLC and so forth.